client/src/components/ProtectedRoute/index.tsx

```diff
@@ -0,0 +1,20 @@
+import { CircularProgress } from '@material-ui/core';
+import { Redirect, Route, RouteProps } from 'react-router-dom';
+import { useAuth } from '../../context/useAuthContext';
+
+interface PrivateRouteProps extends RouteProps {
+  // tslint:disable-next-line:no-any
+  component: any;
```

Comment:
You make this `any` but in the parameters you say `component` is of type `Component`. Why do you need to explicitly define this as `any`?

Reply:
I am not saying type of compoennt is Component, it is renaming variable. And I had to make it any because I could not find a solution. It does not accept, solutions in forums suggested it putting any type.